packages/rocketchat-theme/client/imports/base.css

```diff
@@ -3035,7 +3035,7 @@ body:not(.is-cordova) {
 
 .flex-tab-container {
 	display: flex;
-	flex: 0 0 41px;
+	/*flex: 0 0 41px;*/
```

Comment:
@vickyokrm this is the actual fix, right? Have you tested other browsers?

Reply:
Tested in the chrome and IE.

Reply:
@vickyokrm what about using the `-ms-flex` and `webkit-flex` (see https://www.w3schools.com/cssref/css3_pr_flex.asp)

Reply:
@mrsimpson, machi, -ms-flex was supported until IE 10. From IE 11, the flex property should work without any issues. Whereas in our case, although the flex-basis property was fixed to 41px, webkit recomputes the flex-basis value depending on the width of the child flex items, while IE fails to do that which leads to no room for the dynamic overview screen. An another approach to make this work in IE will be by setting the 'flex-basis' to 'auto'.

Reply:
@vickyokrm `flex: 0 0 auto` is equal to "flex: none". 
Anyway, I'll just merge your PR, please discuss the flexing with the RC core team (at https://github.com/RocketChat/Rocket.Chat/issues/8204) so that we can understand why they wanted to have a basis of 41px - it surely has got some reason

Reply:
@mrsimpson The fixes to handle the multiple duplicate scroll bar issue in the side nav bar is resolved. Seeking your assistance to review it.